doc/web-doc/src/routes/about/+page.svelte

```diff
@@ -0,0 +1,10 @@
+<svelte:head>
+	<title>About</title>
+	<meta name="description" content="About this app" />
+</svelte:head>
+
+<div class="text-column">
+	<h1>About Theia Trace Extension</h1>
+
+	<p>About Theia Trace Extension</p>
```

Comment:
I'd not duplicate the header sentence from above, but rather replace this paragraph with a description, or refer/link to an existing one that is authoritative enough.

Reply:
Done.